R/avgdist.R

```diff
@@ -11,7 +11,10 @@
         transf <- match.fun(transf)
     # Get the list of iteration matrices
     distlist <- lapply(seq_len(iterations), function(i) {
-        inputcast <- rrarefy(x, sample = sample)
```

Comment:
This fails examples: no `inputcast` found. You removed it from this line, and reference it later.